content/docs/guides/branching-neon-cli.md

```diff
@@ -0,0 +1,96 @@
+
+```bash
+neonctl branches list
+┌────────────────────────────┬───────────────────────┬─────────┬──────────────────────┬──────────────────────┐
+│ Id                         │ Name                  │ Primary │ Created At           │ Updated At           │
+├────────────────────────────┼───────────────────────┼─────────┼──────────────────────┼──────────────────────┤
+│ br-lucky-mud-08878834      │ br-lucky-mud-08878834 │ false   │ 2023-07-24T20:22:42Z │ 2023-07-24T20:38:34Z │
+├────────────────────────────┼───────────────────────┼─────────┼──────────────────────┼──────────────────────┤
+│ br-wandering-king-30669552 │ main                  │ true    │ 2023-07-24T15:31:03Z │ 2023-07-24T16:13:48Z │
+└────────────────────────────┴───────────────────────┴─────────┴──────────────────────┴──────────────────────┘
+```
+
+## Delete a branch with the CLI
+
+The following Neon CLI command deletes the specified branch. If your Neon account has more than one project, you will be required to specify a project ID using the `--project-id` option. To view the CLI documentation for this command, refer to the [Neon CLI reference](https://neon.tech/docs/reference/cli-branches#delete). You can delete a branch by its ID or name.
+
+```bash
+neonctl branches delete br-rough-sky-158193
+┌───────────────────────┬───────────────────────┬─────────┬──────────────────────┬──────────────────────┐
+│ Id                    │ Name                  │ Primary │ Created At           │ Updated At           │
+├───────────────────────┼───────────────────────┼─────────┼──────────────────────┼──────────────────────┤
+│ br-lucky-mud-08878834 │ br-lucky-mud-08878834 │ false   │ 2023-07-24T20:22:42Z │ 2023-07-24T20:44:51Z │
+└───────────────────────┴───────────────────────┴─────────┴──────────────────────┴──────────────────────┘
+```
+
+## Branching automation with the Neon CLI
+
+The Neon CLI enables easy automation of branching operations for integration into your workflows or toolchains. To facilitate authentication to Neon when running a CLI command, the Neon CLI provides an `--api-key` option that you can specify at the command line. For information about obtaining an API key, see [Create an API key](/docs/manage/api-keys#create-an-api-key).
+
+To use an API key, you can store it in an environment variable on your system. This prevents the key from being hardcoded into your automation scripts or exposed in another way. For example, you can add the following line to your shell's profile file (`.bashrc` or `.bash_profile` for bash shell):
+
+```bash
+export NEON_API_KEY=<neon_api_key>
+```
+
+After exporting your key, source the profile file (source `~/.bashrc` or source `~/.bash_profile`), or start a new terminal session.
+
+In your scripts or automation tooling, you can reference the environment variable like so:
+
+```bash
+neonctl branches create --api-key $NEON_API_KEY
```

Comment:
no need to pass it directly, NEON_API_KEY env is the default place to look for the API key

Reply:
Thank you. I will fix that.